lms/djangoapps/discussion/templates/discussion/edx_ace/responsenotification/email/body.html

```diff
@@ -7,9 +7,12 @@
     <tr>
         <td>
             <p>
-                {% blocktrans trimmed %}
-                    {{ comment_username }} replied to <b>{{ thread_title }}</b>:
+                {% filter force_escape %}
+                {% blocktrans trimmed asvar replied_to_text %}
+                    {{ comment_username }} replied to
                 {% endblocktrans %}
+                {% endfilter %}
```

Comment:
Please make sure that asvar variable replied_to_text in this case has the escaped string.

Reply:
I have verified it and yes it has the escaped string.